config/locales/en.yml

```diff
@@ -111,6 +112,8 @@ en:
                 for %{product}: %{missing_key}'
               products_do_not_match_manifest_products: Products in manifest (%{manifest_products})
                 do not match products in translations (%{translation_products})
+              insecure_token_parameter_in_manifest: 'Make sure to set secure to true
+                when using keys in Settings. Learn more: %{link}'
```

Comment:
>when requiring keys in Settings

Would probably work better. Since the developer themselves isn't using them, they are requiring them in their app from the customer.